eds/plugin.py

```diff
@@ -1,10 +1,11 @@
 
 from __future__ import annotations
 from typing import Dict, List
+from eds.interfaces.plugin import Plugin
 
 
-class Plugin():
-    """Base class for EDS plugins."""
+class BasePlugin():
```

Comment:
should this be `class BasePlugin(Plugin)` ?

Reply:
yes, you are right.  updated.